packages/backend/routes/auth.ts

```diff
@@ -1,16 +1,34 @@
 import express, { Request, Response, Router } from 'express';
 import passport from 'passport';
+import User from '../models/user/user';
 import { getConfig } from '../config/config';
 
 const config = getConfig();
 const router: Router = express.Router();
 
-router.get('/login/success', (req, res) => {
+interface UserRequest extends Request {
+  user?: any;
```

Comment:
Why `any`?

Reply:
lol fair enough... i can be less lazy here.

Reply:
actually it seems express does not like it when you try and extend user to anything beyond `any`.

Passport even has a `Profile` type declared that is exactly what we want.

```
No overload matches this call.
  The last overload gave the following error.
    Argument of type '(req: UserRequest, res: Response<any, Record<string, any>, number>) => Promise<void>' is not assignable to parameter of type 'Application<Record<string, any>>'.
      Type '(req: UserRequest, res: Response<any, Record<string, any>, number>) => Promise<void>' is missing the following properties from type 'Application<Record<string, any>>': init, defaultConfiguration, engine, set, and 63 more.ts(2769)
```